popover.js

```diff
@@ -40,12 +45,17 @@ Popover.prototype.show = Popover.prototype.render = function(className) {
   setTimeout(function() {
     self.el.classList.add(className || 'show')
   }, 0)
+
+  this.emit('show')
```

Comment:
I think it should be called `shown` and be moved into the `setTimeout` callback, otherwise it'll be called before the the class gets added. What do you think?